st/clitests/awss3api_spec.py

```diff
@@ -164,6 +164,23 @@ def delete_object_list_file(file_name):
 delete_object_list_file(hierarchical_object_list_file)
 
 
+# ************ CopyObject API supported *************************
+AwsTest('Aws can put object').put_object("seagatebucket", "1kfile", 1024)\
+    .execute_test().command_is_successful()
+
+AwsTest('Aws can copy object').copy_object("seagatebucket/1kfile", "seagatebucket", "1kfile-copy")\
+    .execute_test().command_is_successful().command_response_should_have("COPYOBJECTRESULT")
+
+AwsTest('Aws cannot copy object to non-existing destination bucket')\
```

Comment:
Few more to add
1. Copyobject non existent source bucket and source object
2. Copy object from source object to already existing destination bucket and object

Reply:
I will add these system tests once we add source metadata load code.